### DISCUSSION TITLE
Pocketbase is not updating

### DISCUSSION THREAD

**Discussion by sabikrahat** [pocketbase/pocketbase#2589] [Q&A] [answered]
Getting error whenever I try to update pocketbase version 16.2 to 16.3 in windows system. Any suggestion?

`
Fetching release information...
Downloading pocketbase_0.16.3_windows_amd64.zip...
Extracting pocketbase_0.16.3_windows_amd64.zip...
Replacing the executable...
Failed replacing the executable: rename pb_data/.pb_temp_to_delete/extracted_pocketbase_0.16.3_windows_amd64.zip/pocketbase C:\src\pocketbase\pocketbase.exe: The system cannot find the file specified.
`

![image](https://github.com/pocketbase/pocketbase/assets/49031412/71b7a143-d793-4b40-8854-9c2036b389fd)

**Comment by ganigeorgiev** [comment]
Hm, I haven't tested it on Windows. It seems that the extracted executable path is incorrect (missing .exe). I'll setup a Windows VM sometime later next week and will try to investigate it.

For now you can try to replace the binary manually.

**Reply by sabikrahat** [reply]
Okay..

**Reply by ganigeorgiev** [reply, marked as answer]
I've submitted a fix in `master` and tested it in Windows 10 22H2. The fix will be available with the next minor v0.16.4 release sometime next week (_note that the selfupdate on Windows will work from v0.16.4+ onwards, aka. you still will have to update manually one more time from v0.16.3 to v0.16.4_).

**Reply by sabikrahat** [reply]
oh, it's okay..... Thank you so much. @ganigeorgiev
